registrations/tasks.py

```diff
@@ -43,3 +44,41 @@ def send_registration_to_openhim(
     )
     response.raise_for_status()
     return (response.status_code, response.headers, response.content)
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded),
```

Comment:
Maybe we also want to catch the `TembaException`s here, similar to how we did it in forms.py